pkg/util/util.go

```diff
@@ -65,6 +71,25 @@ func GetUUID() string {
 	return string(uuid.NewUUID())
 }
 
+func ReadConfigData(kubeClient kubeclientset.Interface, configMapRef *v1.ObjectReference) string {
```

Comment:
should there be an `err` as part of the func return? Otherwise how does caller know the difference between an empty CM and a GET or Marshal error?

Reply:
Fixed. Thanks for the review.